docs/Helix.md

```diff
@@ -49,7 +49,16 @@ dotnet vstest My.Tests.dll
 If that doesn't help, you can try the Get Repro environment link from mission control and try to debug that way.
 
 ## Differences from running tests locally
-Most tests that don't just work on helix automatically are ones that depend on the source code being accessible. The helix payloads only contain whatever is in the publish directories, so any thing else that test depends on will need to be included to the payload (TBD how to do this).
+Most tests that don't just work on helix automatically are ones that depend on the source code being accessible. The helix payloads only contain whatever is in the publish directories, so any thing else that test depends on will need to be included to the payload.
+
```

Comment:
Might be worth mentioning that HelixContent files will be placed at the root of the directory by default, I'm not exactly sure what the syntax is for specifying a different location, but its either Link or LinkBase (for wildcards).

Reply:
I think its just an additional attribute like `Link="somedir\newName.foo"`

Reply:
Done!